addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -830,10 +848,16 @@ MockServer.include({
             if (!lastMessage) {
                 continue;
             }
+            // deprecated: update mail.channel/fetched_message_id
```

Comment:
mail.channel.partner?

Reply:
same remark, it's refers to a fake field on `mail.channel`.
This is just backward-compatibility for behaviour that still rely on this fake field.

This is to not force ourselves to adapt them right now, which could slow down the making of this PR, which is a requirement for many other PRs.

Reply:
Ok, wording is a bit confusing, usually with deprecated notice we tell what should be used instead, rather than what is wrong. This looks more like a todo.

Reply:
Not sure if the comment is worth it then.

Reply:
Just added comment below to state what we should use.
Let me know if you prefer this or just remove comment (or another suggestion for comment, or something else?)